docs/docs/resources/examples/pipeline.md

```diff
@@ -11,13 +11,10 @@
 
 <!-- ## [Word-count Pipeline](https://github.com/bakdata/kpops-examples/tree/main/word-count/deployment/kpops) -->
 ## Word Count Pipeline
+
+<!-- Replace with https://raw.githubusercontent.com/bakdata/kpops-examples/main/word-count/deployment/kpops/pipeline.yaml? -->
```

Comment:
any issues with that? sounds like a decent plan to me

Reply:
It works!